linker.js

```diff
@@ -1,3 +1,5 @@
+var sha3 = require('crypto-js/sha3');
```

Comment:
This should use `ethereumjs-util`, which is already in `package.json`.

Reply:
ethereumjs-util only is a dev dependency, but I think this here needs to be an actual dependency. I want to avoid pulling in all of ethereumjs-util into the dependencies.

Reply:
Then it should use `keccak.js` (or whatever `ethereumjs-util` is using). `crypto-js/sha3` isn't the best library.